### PR TITLE
440 translate privacy

### PR DESCRIPTION
**Pull request by studiozandra**
Feature added for [issue #440](https://github.com/JWManagement/JWManagement/issues/440):
===

  1. **English Privacy Policy:** machine translation via DeepL.com -- english has been proofread by me.
     **Tested by changing browser language in Chrome on Linux :** the correct URL matching the browser language is shown 

Peek 2022-07-11optimized

gif made using https://github.com/phw/peek 
and 
https://ezgif.com/optimize

### CODE REVIEW

**Comment by tim-antkowiak** [comment]
Hey @studiozandra thank you so much for contributing!

We made some changes yesterday on the develop branch, sorry for the confusion.

To fix the commits you need to rebase your branch again on the current develop branch:

```
# Add the remote, call it "upstream":
git remote add upstream git@github.com:JWManagement/JWManagement.git

# Fetch all the branches of that remote into remote-tracking branches
git fetch upstream

# Make sure that you're on your branch:
git checkout 440-translate-privacy

# Rewrite your branch so that any commits of yours that
# aren't already in upstream/master are replayed on top of that
# other branch:
git rebase upstream/develop

# (Force) Push your branch
git push origin 440-translate-privacy --force
```

Could you try these steps?

**Comment by studiozandra** [comment]
Thank you very kindly! I have run those commands. Still a couple of other commits in there 🤔

**Comment by tim-antkowiak** [comment]
Yea, that were the commits we removed from the develop branch yesterday. I just updated your branch. 
Make sure to do a `git fetch` and maybe a `git reset --hard origin/440-translate-privacy` to get the updated branch on your local setup.

File: client/views/common/footer/footer.tpl.jade [hunk 1]
```diff
@@ -8,4 +8,4 @@
 	div
 		a(href='/{{getLanguage}}/terms' target='blank') Terms
 		| |
-		a(href='/de/privacy' target='blank') Datenschutzerklärung
+		a(href='/{{getLanguage}}/privacy' target='blank') Privacy
```
[the remark this answers is not in the document]
Comment (tim-antkowiak) [reply client/views/common/footer/footer.tpl.jade:11]:
```suggestion
		a(href='/{{getLanguage}}/privacy' target='blank') {{_ "signUp.privacyPolicy"}}
```

File: client/views/common/footer/footer.tpl.jade [hunk 2]
```diff
@@ -8,4 +8,4 @@
 	div
 		a(href='/{{getLanguage}}/terms' target='blank') Terms
```
Comment (tim-antkowiak) [inline client/views/common/footer/footer.tpl.jade:9]:
```suggestion
		a(href='/{{getLanguage}}/terms' target='blank') {{_ "signUp.terms"}}
```

**Review by tim-antkowiak — CHANGES_REQUESTED** [review]

**Review by tim-antkowiak — APPROVED** [review]

**Pull request merged**